recipes/dpp/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+import os
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Git
+from conan.tools.files import download, unzip
+
+required_conan_version = ">=1.65"
+
+class DPPConan(ConanFile):
+    name = "dpp"
+    license = "Apache-2.0"
+    package_type = "shared-library"
```

Comment:
If this is alwas a shared library, then we should not have a `shared` option either (Or the other way around! Set this to `library` to allow both static and shared variants)